.circleci/config.yml

```diff
@@ -41,4 +45,9 @@ jobs:
 workflows:
   default-workflow:
     jobs:
-      - tests
+      - tests:
+          matrix:
+            parameters:
+              wordpress-version:
+                - '5.6.2'
```

Comment:
How do version upgrades work on WP?  Do we stay on 5.6.2 until we manually change it to a different version?

Reply:
> How do version upgrades work on WP? Do we stay on 5.6.2 until we manually change it to a different version?

I think so, yes.

Reply:
Cool.  We just need to remember to update this file at the same time then.

Reply:
It's not clear how this will work actually. Our instance isn't fully managed, so we will have to do the upgrades.

Having latest in CI should help a bit here, hopefully.